docs/release-notes/dictionary-releases.md

```diff
@@ -19,6 +19,38 @@ Coming soon.
 None to report.
 ------>
 
+## Release 1.8
+
+**Release Date:** April 27, 2021
+
+### Updates
+Two **new** data type tables have been added to the dictionary for submission:
+1. **Immunotherapy**: For treatment regimens that include immunotherapy, these details can now be submitted.  Similar to the `chemotherapy` and `hormone_therapy`, `immunotherapy drugs are recorded using the standardized vocabulary from the [RxNorm database](https://www.nlm.nih.gov/research/umls/rxnorm), provided by the NIH. 
```

Comment:
there's an extra single quote in front of immunotherapy (you maybe want a closing single quote?)